web/ts/create-index.ts

```diff
@@ -1,17 +1,15 @@
-const MiniSearch = require('minisearch');
 import { promises as fs } from 'fs';
 import * as path from 'path';
+const MiniSearch = require('minisearch');
```

Comment:
mixing import and require?

Reply:
for some reason it does not work with import. didn't want to burn time debugging.